hack/.custom-gcl.yaml

```diff
@@ -0,0 +1,6 @@
+version:  v1.64.8
```

Comment:
This is the latest version of the v1 series. K/K currently uses 1.64.5, we may want this to match, but it seems to work even on the newer patch version

When golangci-lint is upgraded to v2, we will also need to upgrade here as well.